src/Proto.Cluster/DefaultClusterContext.cs

```diff
@@ -36,6 +37,8 @@ public DefaultClusterContext(IIdentityLookup identityLookup, PidCache pidCache,
                 _config.RequestLogThrottlePeriod,
                 i => _logger.LogInformation("Throttled {LogCount} TryRequestAsync logs", i)
             );
+            _clock = new TaskClock(config.ActorRequestTimeout, TimeSpan.FromSeconds(1), killSwitch);
```

Comment:
Perhaps make sure that the chosen timeout is at least ActorRequestTimeout, by adding the updateInterval to the timeout delays.